### PR TITLE
프로그래머스 최댓값, 최솟값 문제 풀이

### PR DESCRIPTION
**Pull request by Kimdongui**
<h6 class="guide-section-title" style="box-sizing: border-box; padding: 0px; margin: 0px 0px 1rem; color: white; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-variant-numeric: inherit; font-variant-east-asian: inherit; font-weight: 700; font-stretch: inherit; font-size: 14px; line-height: 1.5; font-family: Inter, NotoSansKR, &quot;Helvetica Neue&quot;, Helvetica, Arial, &quot;맑은 고딕&quot;, &quot;malgun gothic&quot;, 돋움, Dotum, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Noto Color Emoji&quot;; -webkit-font-smoothing: antialiased; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(38, 55, 71); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">문제 설명</h6><div class="markdown solarized-dark" style="box-sizing: border-box; padding: 0px; margin: 0px; color: rgb(178, 192, 204); font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-variant-numeric: inherit; font-variant-east-asian: inherit; font-weight: 400; font-stretch: inherit; font-size: 16px; line-height: inherit; font-family: Inter, NotoSansKR, &quot;Helvetica Neue&quot;, Helvetica, Arial, &quot;맑은 고딕&quot;, &quot;malgun gothic&quot;, 돋움, Dotum, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Noto Color Emoji&quot;; background-color: rgb(38, 55, 71); word-break: normal; overflow-wrap: break-word; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><p style="box-sizing: border-box; padding: 0px; margin: 0px; color: inherit; font-style: inherit; font-variant: inherit; font-weight: 400; font-stretch: inherit; font-size: inherit; line-height: 1.8; font-family: Inter, NotoSansKR, &quot;Helvetica Neue&quot;, Helvetica, Arial, &quot;맑은 고딕&quot;, &quot;malgun gothic&quot;, 돋움, Dotum, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Noto Color Emoji&quot;; letter-spacing: -0.1px; overflow-wrap: break-word;">문자열 s에는 공백으로 구분된 숫자들이 저장되어 있습니다. str에 나타나는 숫자 중 최소값과 최대값을 찾아 이를 "(최소값) (최대값)"형태의 문자열을 반환하는 함수, solution을 완성하세요.<br style="box-sizing: border-box; padding: 0px; margin: 0px; color: inherit; font: inherit;">예를들어 s가 "1 2 3 4"라면 "1 4"를 리턴하고, "-1 -2 -3 -4"라면 "-4 -1"을 리턴하면 됩니다.</p><h5 style="box-sizing: border-box; padding: 0px; margin: 1.5rem 0px 0px; color: white; font-style: inherit; font-variant: inherit; font-weight: 400; font-stretch: inherit; font-size: 15px; line-height: 1.5; font-family: Inter, NotoSansKR, &quot;Helvetica Neue&quot;, Helvetica, Arial, &quot;맑은 고딕&quot;, &quot;malgun gothic&quot;, 돋움, Dotum, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Noto Color Emoji&quot;; letter-spacing: -0.009em;">제한 조건</h5><ul style="box-sizing: border-box; padding: 0px 0px 0px 2.5rem; margin: 0.75rem 0px 0px; color: inherit; font: inherit;"><li style="box-sizing: border-box; padding: 0px; margin: 0px; color: inherit; font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; font-size: 16px; line-height: 1.6; font-family: inherit; letter-spacing: -0.009em; list-style: inherit;">s에는 둘 이상의 정수가 공백으로 구분되어 있습니다.</li></ul><h5 style="box-sizing: border-box; padding: 0px; margin: 1.5rem 0px 0px; color: white; font-style: inherit; font-variant: inherit; font-weight: 400; font-stretch: inherit; font-size: 15px; line-height: 1.5; font-family: Inter, NotoSansKR, &quot;Helvetica Neue&quot;, Helvetica, Arial, &quot;맑은 고딕&quot;, &quot;malgun gothic&quot;, 돋움, Dotum, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Noto Color Emoji&quot;; letter-spacing: -0.009em;">입출력 예</h5>

s | return
-- | --
"1 2 3 4" | "1 4"
"-1 -2 -3 -4" | "-4 -1"
"-1 -1" | "-1 -1"

</div>

### CODE REVIEW

**Review by sxs770 — DISMISSED** [review]

**Review by Chohyungun — APPROVED** [review]
으아아

File: programmers/최솟값 최댓값.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"""
+작성자 : donghyun Kim
+문제 : https://school.programmers.co.kr/learn/courses/30/lessons/12939
+"""
+
+def solution(s):
+    l = list(map(int, s.split()))
+    return "{} {}".format(min(l), max(l))
```
Comment (LHyunn) [inline programmers/최솟값 최댓값.py]: 실제로 회사에서 코테 면접을 볼때 min max 같은 메소드들을 사용해도 되나요? sort()같은..? 상관없나요? 아 얘는 리스트의 메소드들을 잘 알고있구나? 라고 생각하나요 아니면 편법쓰네 같은 생각인지..?

File: programmers/최솟값 최댓값.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+"""
+작성자 : donghyun Kim
+문제 : https://school.programmers.co.kr/learn/courses/30/lessons/12939
+"""
+
+def solution(s):
+    l = list(map(int, s.split()))
```
Comment (ChangZero) [inline programmers/최솟값 최댓값.py]: 변수명이 무엇을 뜻하는지 잘 모르겠습니다.

File: programmers/최솟값 최댓값.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+"""
+작성자 : donghyun Kim
+문제 : https://school.programmers.co.kr/learn/courses/30/lessons/12939
+"""
+
```
Comment (CodeofO) [inline programmers/최솟값 최댓값.py:5]: 깃깃깃

**Review by YexeoKim — APPROVED** [review]
좋은 코드인 것 같습니다.

**Review by ChangZero — APPROVED** [review]

**Review by sxs770 — APPROVED** [review]

File: programmers/최솟값 최댓값.py (hunk 2 above)
Comment (Kimdongui) [reply programmers/최솟값 최댓값.py]: 오.. 감사합니다 반영해서 고쳐봤습니다!

File: programmers/최솟값 최댓값.py (hunk 1 above)
Comment (Kimdongui) [reply programmers/최솟값 최댓값.py]: 네 크게 상관없습니다.

**Pull request merged**